### ISSUE TITLE
Main node can not inspect the state of the cluster

### ISSUE THREAD

**Issue by javiyu** [memgraph/memgraph#1575]
**Memgraph version**
v2.13.0

**Environment**
Running on a kubernetes cluster, a memgraph cluster setup with 3 nodes, 1 main and 2 slaves.

**Describe the bug**
After some time with almost no activity on the cluster, the MAIN seems to crash

```
$ kubectl get pods -n preview-memgraph
NAME                                           READY   STATUS      RESTARTS       AGE
memgraph-0                                     1/1     Running     11 (14h ago)   14h
memgraph-1                                     1/1     Running     0              14h
memgraph-2                                     1/1     Running     0              19m
```

Not sure about the reason but I am when I open a console on the main node ( memgraph-0 ) and I try to run a simple `show replicas` the query get stuck and it never finishes.

**Expected behavior**
The node to be able to report the state of the cluster.

**Comment by antejavor** [comment]
@andrejtonev and @Ignition, just a context @javiyu is testing our v2.13 replication in his setup.

**Comment by andrejtonev** [comment]
Hi @javiyu do you have any logs from the MAIN or REPLICAS?
Thanks!

**Comment by DavIvek** [comment]
@javiyu Were you able to run other queries, for example, create or match? Was the show replicas only the one that was causing the problems?
Did you use DNS names to register replicas, or did you use IP addresses?

**Comment by javiyu** [comment]
> @javiyu Were you able to run other queries, for example, create or match? Was the show replicas only the one that was causing the problems? Did you use DNS names to register replicas, or did you use IP addresses?

@DavIvek the node would not respond to any query, I was using DNS names for registering the replicas.

**Comment by DavIvek** [comment]
@javiyu We had some problems with DNS, that should be resolved in next release.

**Comment by DavIvek** [comment]
This should be fixed with #1615 .

**Comment by karmenrabar** [comment]
Hi @javiyu, I'm happy to update you that Memgraph has a new release, 2.14., which includes the fix for this issue. We post all of our release updates on Discord, here's the [link](https://discord.com/channels/842007348272169002/1047507324555767858/1200107892968271956) - feel free to join if you already haven't. :)
Could you check if the newest version works as expected for you now?
